### ISSUE TITLE
Pre-work Checklist: Developer: Belen Garcia Martinez

### ISSUE THREAD

**Issue by 93Belen** [hackforla/website#4631]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by 93Belen** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings? 5 hours so far (4 hours meeting last night, one hour working on pre-work today)

**Comment by 93Belen** [comment]
Progress: I have completed all the steps of this issue
Blockers: None
Availability: Sunday from 6am to 6pm
ETA: 5/19/23

**Comment by HackforLABot** [comment]
## Activity Log: 93Belen

#####  ⚠ Important note: The bot updates this issue automatically - do not edit

- 93Belen opened issue: [4631](https://github.com/hackforla/website/issues/4631) at 2023-05-08 08:35 PM PDT
- 93Belen assigned to issue: [4631](https://github.com/hackforla/website/issues/4631) at 2023-05-08 08:45 PM PDT
- 93Belen assigned to issue: [4473](https://github.com/hackforla/website/issues/4473#issuecomment-1504472218) at 2023-05-09 07:11 AM PDT
- 93Belen opened pull request: [4643](https://github.com/hackforla/website/pull/4643) at 2023-05-09 07:31 AM PDT
- 93Belen commented on issue: [4473](https://github.com/hackforla/website/issues/4473#issuecomment-1540258746) at 2023-05-09 07:39 AM PDT
- 93Belen commented on issue: [4631](https://github.com/hackforla/website/issues/4631#issuecomment-1540266766) at 2023-05-09 07:44 AM PDT
- 93Belen commented on pull request: [4643](https://github.com/hackforla/website/pull/4643#issuecomment-1541889009) at 2023-05-10 03:29 AM PDT
- 93Belen opened pull request: [4646](https://github.com/hackforla/website/pull/4646) at 2023-05-10 03:39 AM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1541911235) at 2023-05-10 03:45 AM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1542705702) at 2023-05-10 12:37 PM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1542724514) at 2023-05-10 12:53 PM PDT
- 93Belen pull request closed w/o merging: [4643](https://github.com/hackforla/website/pull/4643#event-9218030767) at 2023-05-10 10:29 PM PDT
- 93Belen commented on issue: [4473](https://github.com/hackforla/website/issues/4473#issuecomment-1544469877) at 2023-05-11 11:18 AM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1545490853) at 2023-05-12 02:58 AM PDT
- 93Belen pull request closed w/o merging: [4646](https://github.com/hackforla/website/pull/4646#event-9239125435) at 2023-05-13 12:44 PM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1546740008) at 2023-05-13 12:57 PM PDT
- 93Belen opened pull request: [4666](https://github.com/hackforla/website/pull/4666) at 2023-05-14 10:16 AM PDT
- 93Belen commented on pull request: [4666](https://github.com/hackforla/website/pull/4666#issuecomment-1546951524) at 2023-05-14 10:18 AM PDT
- 93Belen commented on pull request: [4665](https://github.com/hackforla/website/pull/4665#issuecomment-1546953338) at 2023-05-14 10:25 AM PDT
- 93Belen submitted pull request review: [4665](https://github.com/hackforla/website/pull/4665#pullrequestreview-1425597241) at 2023-05-14 01:23 PM PDT
- 93Belen assigned to issue: [4642](https://github.com/hackforla/website/issues/4642) at 2023-05-15 01:16 PM PDT
- 93Belen commented on issue: [4642](https://github.com/hackforla/website/issues/4642#issuecomment-1548517786) at 2023-05-15 01:18 PM PDT
- 93Belen opened pull request: [4675](https://github.com/hackforla/website/pull/4675) at 2023-05-15 01:32 PM PDT
- 93Belen commented on pull request: [4675](https://github.com/hackforla/website/pull/4675#issuecomment-1548535428) at 2023-05-15 01:33 PM PDT
- 93Belen pull request closed w/o merging: [4675](https://github.com/hackforla/website/pull/4675#event-9250866880) at 2023-05-15 01:34 PM PDT
- 93Belen opened pull request: [4677](https://github.com/hackforla/website/pull/4677) at 2023-05-15 01:56 PM PDT
- 93Belen commented on pull request: [4677](https://github.com/hackforla/website/pull/4677#issuecomment-1548564909) at 2023-05-15 01:57 PM PDT
- 93Belen commented on pull request: [4666](https://github.com/hackforla/website/pull/4666#issuecomment-1549571222) at 2023-05-16 05:29 AM PDT
- 93Belen commented on pull request: [4674](https://github.com/hackforla/website/pull/4674#issuecomment-1549677168) at 2023-05-16 06:27 AM PDT
- 93Belen submitted pull request review: [4674](https://github.com/hackforla/website/pull/4674#pullrequestreview-1428600446) at 2023-05-16 06:34 AM PDT
- 93Belen pull request merged: [4677](https://github.com/hackforla/website/pull/4677#event-9263804513) at 2023-05-16 06:43 PM PDT
- 93Belen pull request merged: [4666](https://github.com/hackforla/website/pull/4666#event-9263822406) at 2023-05-16 06:47 PM PDT
- 93Belen commented on pull request: [4674](https://github.com/hackforla/website/pull/4674#issuecomment-1550593266) at 2023-05-16 07:22 PM PDT
- 93Belen assigned to issue: [4687](https://github.com/hackforla/website/issues/4687) at 2023-05-17 03:43 PM PDT
- 93Belen commented on issue: [4687](https://github.com/hackforla/website/issues/4687#issuecomment-1552179712) at 2023-05-17 03:45 PM PDT
- 93Belen opened pull request: [4691](https://github.com/hackforla/website/pull/4691) at 2023-05-18 03:58 AM PDT
- 93Belen commented on pull request: [4689](https://github.com/hackforla/website/pull/4689#issuecomment-1552899785) at 2023-05-18 04:09 AM PDT
- 93Belen submitted pull request review: [4689](https://github.com/hackforla/website/pull/4689#pullrequestreview-1432518804) at 2023-05-18 04:21 AM PDT
- 93Belen commented on pull request: [4691](https://github.com/hackforla/website/pull/4691#issuecomment-1553520661) at 2023-05-18 12:13 PM PDT
- 93Belen pull request merged: [4691](https://github.com/hackforla/website/pull/4691#event-9286175614) at 2023-05-18 11:30 PM PDT
- 93Belen commented on issue: [4631](https://github.com/hackforla/website/issues/4631#issuecomment-1555132044) at 2023-05-19 12:26 PM PDT
- 93Belen commented on pull request: [4702](https://github.com/hackforla/website/pull/4702#issuecomment-1556227024) at 2023-05-21 10:07 AM PDT
- 93Belen assigned to issue: [4662](https://github.com/hackforla/website/issues/4662) at 2023-05-22 04:19 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1557040450) at 2023-05-22 04:20 AM PDT
- 93Belen submitted pull request review: [4702](https://github.com/hackforla/website/pull/4702#pullrequestreview-1439648024) at 2023-05-23 06:00 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1559601481) at 2023-05-23 07:54 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1559621561) at 2023-05-23 08:01 AM PDT
- 93Belen commented on pull request: [4702](https://github.com/hackforla/website/pull/4702#issuecomment-1560011712) at 2023-05-23 12:28 PM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1560377117) at 2023-05-23 07:37 PM PDT
- 93Belen commented on pull request: [4710](https://github.com/hackforla/website/pull/4710#issuecomment-1560379555) at 2023-05-23 07:41 PM PDT
- 93Belen submitted pull request review: [4702](https://github.com/hackforla/website/pull/4702#pullrequestreview-1444004597) at 2023-05-25 07:16 AM PDT
- 93Belen submitted pull request review: [4710](https://github.com/hackforla/website/pull/4710#pullrequestreview-1444081720) at 2023-05-25 07:51 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1564204141) at 2023-05-26 03:49 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1564824546) at 2023-05-26 12:19 PM PDT
- 93Belen commented on pull request: [4746](https://github.com/hackforla/website/pull/4746#issuecomment-1564979890) at 2023-05-26 02:29 PM PDT
- 93Belen submitted pull request review: [4746](https://github.com/hackforla/website/pull/4746#pullrequestreview-1447000349) at 2023-05-26 02:45 PM PDT
- 93Belen submitted pull request review: [4746](https://github.com/hackforla/website/pull/4746#pullrequestreview-1447027631) at 2023-05-26 03:35 PM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1565336598) at 2023-05-27 03:27 AM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1566214809) at 2023-05-28 11:05 AM PDT
- 93Belen commented on pull request: [4732](https://github.com/hackforla/website/pull/4732#issuecomment-1566214953) at 2023-05-28 11:05 AM PDT
- 93Belen submitted pull request review: [4731](https://github.com/hackforla/website/pull/4731#pullrequestreview-1448373188) at 2023-05-28 12:01 PM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1566231274) at 2023-05-28 12:03 PM PDT
- 93Belen commented on pull request: [4732](https://github.com/hackforla/website/pull/4732#issuecomment-1567291890) at 2023-05-29 08:46 AM PDT
- 93Belen submitted pull request review: [4732](https://github.com/hackforla/website/pull/4732#pullrequestreview-1450918277) at 2023-05-30 06:18 AM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1568551575) at 2023-05-30 07:37 AM PDT
- 93Belen commented on pull request: [4754](https://github.com/hackforla/website/pull/4754#issuecomment-1568557089) at 2023-05-30 07:40 AM PDT
- 93Belen submitted pull request review: [4754](https://github.com/hackforla/website/pull/4754#pullrequestreview-1451144991) at 2023-05-30 07:46 AM PDT
- 93Belen closed issue as completed: [4631](https://github.com/hackforla/website/issues/4631#event-9383291032) at 2023-05-30 02:48 PM PDT
- 93Belen commented on pull request: [4755](https://github.com/hackforla/website/pull/4755#issuecomment-1569409357) at 2023-05-30 07:34 PM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1454219190) at 2023-05-31 02:41 PM PDT
- 93Belen commented on pull request: [4759](https://github.com/hackforla/website/pull/4759#issuecomment-1571031533) at 2023-05-31 03:11 PM PDT
- 93Belen submitted pull request review: [4759](https://github.com/hackforla/website/pull/4759#pullrequestreview-1455548062) at 2023-06-01 06:57 AM PDT
- 93Belen assigned to issue: [4525](https://github.com/hackforla/website/issues/4525#issuecomment-1575580469) at 2023-06-04 09:38 AM PDT
- 93Belen unassigned from issue: [4525](https://github.com/hackforla/website/issues/4525#issuecomment-1575633125) at 2023-06-04 09:46 AM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1465261893) at 2023-06-06 06:49 AM PDT
- 93Belen commented on pull request: [4783](https://github.com/hackforla/website/pull/4783#issuecomment-1578838212) at 2023-06-06 07:07 AM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1466176435) at 2023-06-06 02:52 PM PDT
- 93Belen submitted pull request review: [4783](https://github.com/hackforla/website/pull/4783#pullrequestreview-1466297720) at 2023-06-06 04:35 PM PDT
- 93Belen commented on pull request: [4755](https://github.com/hackforla/website/pull/4755#issuecomment-1579601707) at 2023-06-06 04:39 PM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1466301154) at 2023-06-06 04:40 PM PDT
- 93Belen commented on pull request: [4755](https://github.com/hackforla/website/pull/4755#issuecomment-1579603412) at 2023-06-06 04:41 PM PDT
- 93Belen submitted pull request review: [4783](https://github.com/hackforla/website/pull/4783#pullrequestreview-1468533228) at 2023-06-07 01:43 PM PDT
- 93Belen submitted pull request review: [4783](https://github.com/hackforla/website/pull/4783#pullrequestreview-1474827231) at 2023-06-12 05:36 AM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1627773113) at 2023-07-09 09:59 AM PDT
- 93Belen submitted pull request review: [4731](https://github.com/hackforla/website/pull/4731#pullrequestreview-1525194986) at 2023-07-11 02:39 PM PDT

**Comment by HackforLABot** [comment]
## Activity Log: 93Belen

#####  ⚠ Important note: The bot updates this issue automatically - do not edit

- 93Belen opened issue: [4631](https://github.com/hackforla/website/issues/4631) at 2023-05-08 08:35 PM PDT
- 93Belen assigned to issue: [4631](https://github.com/hackforla/website/issues/4631) at 2023-05-08 08:45 PM PDT
- 93Belen assigned to issue: [4473](https://github.com/hackforla/website/issues/4473#issuecomment-1504472218) at 2023-05-09 07:11 AM PDT
- 93Belen opened pull request: [4643](https://github.com/hackforla/website/pull/4643) at 2023-05-09 07:31 AM PDT
- 93Belen commented on issue: [4473](https://github.com/hackforla/website/issues/4473#issuecomment-1540258746) at 2023-05-09 07:39 AM PDT
- 93Belen commented on issue: [4631](https://github.com/hackforla/website/issues/4631#issuecomment-1540266766) at 2023-05-09 07:44 AM PDT
- 93Belen commented on pull request: [4643](https://github.com/hackforla/website/pull/4643#issuecomment-1541889009) at 2023-05-10 03:29 AM PDT
- 93Belen opened pull request: [4646](https://github.com/hackforla/website/pull/4646) at 2023-05-10 03:39 AM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1541911235) at 2023-05-10 03:45 AM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1542705702) at 2023-05-10 12:37 PM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1542724514) at 2023-05-10 12:53 PM PDT
- 93Belen pull request closed w/o merging: [4643](https://github.com/hackforla/website/pull/4643#event-9218030767) at 2023-05-10 10:29 PM PDT
- 93Belen commented on issue: [4473](https://github.com/hackforla/website/issues/4473#issuecomment-1544469877) at 2023-05-11 11:18 AM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1545490853) at 2023-05-12 02:58 AM PDT
- 93Belen pull request closed w/o merging: [4646](https://github.com/hackforla/website/pull/4646#event-9239125435) at 2023-05-13 12:44 PM PDT
- 93Belen commented on pull request: [4646](https://github.com/hackforla/website/pull/4646#issuecomment-1546740008) at 2023-05-13 12:57 PM PDT
- 93Belen opened pull request: [4666](https://github.com/hackforla/website/pull/4666) at 2023-05-14 10:16 AM PDT
- 93Belen commented on pull request: [4666](https://github.com/hackforla/website/pull/4666#issuecomment-1546951524) at 2023-05-14 10:18 AM PDT
- 93Belen commented on pull request: [4665](https://github.com/hackforla/website/pull/4665#issuecomment-1546953338) at 2023-05-14 10:25 AM PDT
- 93Belen submitted pull request review: [4665](https://github.com/hackforla/website/pull/4665#pullrequestreview-1425597241) at 2023-05-14 01:23 PM PDT
- 93Belen assigned to issue: [4642](https://github.com/hackforla/website/issues/4642) at 2023-05-15 01:16 PM PDT
- 93Belen commented on issue: [4642](https://github.com/hackforla/website/issues/4642#issuecomment-1548517786) at 2023-05-15 01:18 PM PDT
- 93Belen opened pull request: [4675](https://github.com/hackforla/website/pull/4675) at 2023-05-15 01:32 PM PDT
- 93Belen commented on pull request: [4675](https://github.com/hackforla/website/pull/4675#issuecomment-1548535428) at 2023-05-15 01:33 PM PDT
- 93Belen pull request closed w/o merging: [4675](https://github.com/hackforla/website/pull/4675#event-9250866880) at 2023-05-15 01:34 PM PDT
- 93Belen opened pull request: [4677](https://github.com/hackforla/website/pull/4677) at 2023-05-15 01:56 PM PDT
- 93Belen commented on pull request: [4677](https://github.com/hackforla/website/pull/4677#issuecomment-1548564909) at 2023-05-15 01:57 PM PDT
- 93Belen commented on pull request: [4666](https://github.com/hackforla/website/pull/4666#issuecomment-1549571222) at 2023-05-16 05:29 AM PDT
- 93Belen commented on pull request: [4674](https://github.com/hackforla/website/pull/4674#issuecomment-1549677168) at 2023-05-16 06:27 AM PDT
- 93Belen submitted pull request review: [4674](https://github.com/hackforla/website/pull/4674#pullrequestreview-1428600446) at 2023-05-16 06:34 AM PDT
- 93Belen pull request merged: [4677](https://github.com/hackforla/website/pull/4677#event-9263804513) at 2023-05-16 06:43 PM PDT
- 93Belen pull request merged: [4666](https://github.com/hackforla/website/pull/4666#event-9263822406) at 2023-05-16 06:47 PM PDT
- 93Belen commented on pull request: [4674](https://github.com/hackforla/website/pull/4674#issuecomment-1550593266) at 2023-05-16 07:22 PM PDT
- 93Belen assigned to issue: [4687](https://github.com/hackforla/website/issues/4687) at 2023-05-17 03:43 PM PDT
- 93Belen commented on issue: [4687](https://github.com/hackforla/website/issues/4687#issuecomment-1552179712) at 2023-05-17 03:45 PM PDT
- 93Belen opened pull request: [4691](https://github.com/hackforla/website/pull/4691) at 2023-05-18 03:58 AM PDT
- 93Belen commented on pull request: [4689](https://github.com/hackforla/website/pull/4689#issuecomment-1552899785) at 2023-05-18 04:09 AM PDT
- 93Belen submitted pull request review: [4689](https://github.com/hackforla/website/pull/4689#pullrequestreview-1432518804) at 2023-05-18 04:21 AM PDT
- 93Belen commented on pull request: [4691](https://github.com/hackforla/website/pull/4691#issuecomment-1553520661) at 2023-05-18 12:13 PM PDT
- 93Belen pull request merged: [4691](https://github.com/hackforla/website/pull/4691#event-9286175614) at 2023-05-18 11:30 PM PDT
- 93Belen commented on issue: [4631](https://github.com/hackforla/website/issues/4631#issuecomment-1555132044) at 2023-05-19 12:26 PM PDT
- 93Belen commented on pull request: [4702](https://github.com/hackforla/website/pull/4702#issuecomment-1556227024) at 2023-05-21 10:07 AM PDT
- 93Belen assigned to issue: [4662](https://github.com/hackforla/website/issues/4662) at 2023-05-22 04:19 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1557040450) at 2023-05-22 04:20 AM PDT
- 93Belen submitted pull request review: [4702](https://github.com/hackforla/website/pull/4702#pullrequestreview-1439648024) at 2023-05-23 06:00 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1559601481) at 2023-05-23 07:54 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1559621561) at 2023-05-23 08:01 AM PDT
- 93Belen commented on pull request: [4702](https://github.com/hackforla/website/pull/4702#issuecomment-1560011712) at 2023-05-23 12:28 PM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1560377117) at 2023-05-23 07:37 PM PDT
- 93Belen commented on pull request: [4710](https://github.com/hackforla/website/pull/4710#issuecomment-1560379555) at 2023-05-23 07:41 PM PDT
- 93Belen submitted pull request review: [4702](https://github.com/hackforla/website/pull/4702#pullrequestreview-1444004597) at 2023-05-25 07:16 AM PDT
- 93Belen submitted pull request review: [4710](https://github.com/hackforla/website/pull/4710#pullrequestreview-1444081720) at 2023-05-25 07:51 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1564204141) at 2023-05-26 03:49 AM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1564824546) at 2023-05-26 12:19 PM PDT
- 93Belen commented on pull request: [4746](https://github.com/hackforla/website/pull/4746#issuecomment-1564979890) at 2023-05-26 02:29 PM PDT
- 93Belen submitted pull request review: [4746](https://github.com/hackforla/website/pull/4746#pullrequestreview-1447000349) at 2023-05-26 02:45 PM PDT
- 93Belen submitted pull request review: [4746](https://github.com/hackforla/website/pull/4746#pullrequestreview-1447027631) at 2023-05-26 03:35 PM PDT
- 93Belen commented on issue: [4662](https://github.com/hackforla/website/issues/4662#issuecomment-1565336598) at 2023-05-27 03:27 AM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1566214809) at 2023-05-28 11:05 AM PDT
- 93Belen commented on pull request: [4732](https://github.com/hackforla/website/pull/4732#issuecomment-1566214953) at 2023-05-28 11:05 AM PDT
- 93Belen submitted pull request review: [4731](https://github.com/hackforla/website/pull/4731#pullrequestreview-1448373188) at 2023-05-28 12:01 PM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1566231274) at 2023-05-28 12:03 PM PDT
- 93Belen commented on pull request: [4732](https://github.com/hackforla/website/pull/4732#issuecomment-1567291890) at 2023-05-29 08:46 AM PDT
- 93Belen submitted pull request review: [4732](https://github.com/hackforla/website/pull/4732#pullrequestreview-1450918277) at 2023-05-30 06:18 AM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1568551575) at 2023-05-30 07:37 AM PDT
- 93Belen commented on pull request: [4754](https://github.com/hackforla/website/pull/4754#issuecomment-1568557089) at 2023-05-30 07:40 AM PDT
- 93Belen submitted pull request review: [4754](https://github.com/hackforla/website/pull/4754#pullrequestreview-1451144991) at 2023-05-30 07:46 AM PDT
- 93Belen closed issue as completed: [4631](https://github.com/hackforla/website/issues/4631#event-9383291032) at 2023-05-30 02:48 PM PDT
- 93Belen commented on pull request: [4755](https://github.com/hackforla/website/pull/4755#issuecomment-1569409357) at 2023-05-30 07:34 PM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1454219190) at 2023-05-31 02:41 PM PDT
- 93Belen commented on pull request: [4759](https://github.com/hackforla/website/pull/4759#issuecomment-1571031533) at 2023-05-31 03:11 PM PDT
- 93Belen submitted pull request review: [4759](https://github.com/hackforla/website/pull/4759#pullrequestreview-1455548062) at 2023-06-01 06:57 AM PDT
- 93Belen assigned to issue: [4525](https://github.com/hackforla/website/issues/4525#issuecomment-1575580469) at 2023-06-04 09:38 AM PDT
- 93Belen unassigned from issue: [4525](https://github.com/hackforla/website/issues/4525#issuecomment-1575633125) at 2023-06-04 09:46 AM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1465261893) at 2023-06-06 06:49 AM PDT
- 93Belen commented on pull request: [4783](https://github.com/hackforla/website/pull/4783#issuecomment-1578838212) at 2023-06-06 07:07 AM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1466176435) at 2023-06-06 02:52 PM PDT
- 93Belen submitted pull request review: [4783](https://github.com/hackforla/website/pull/4783#pullrequestreview-1466297720) at 2023-06-06 04:35 PM PDT
- 93Belen commented on pull request: [4755](https://github.com/hackforla/website/pull/4755#issuecomment-1579601707) at 2023-06-06 04:39 PM PDT
- 93Belen submitted pull request review: [4755](https://github.com/hackforla/website/pull/4755#pullrequestreview-1466301154) at 2023-06-06 04:40 PM PDT
- 93Belen commented on pull request: [4755](https://github.com/hackforla/website/pull/4755#issuecomment-1579603412) at 2023-06-06 04:41 PM PDT
- 93Belen submitted pull request review: [4783](https://github.com/hackforla/website/pull/4783#pullrequestreview-1468533228) at 2023-06-07 01:43 PM PDT
- 93Belen submitted pull request review: [4783](https://github.com/hackforla/website/pull/4783#pullrequestreview-1474827231) at 2023-06-12 05:36 AM PDT
- 93Belen commented on pull request: [4731](https://github.com/hackforla/website/pull/4731#issuecomment-1627773113) at 2023-07-09 09:59 AM PDT
- 93Belen submitted pull request review: [4731](https://github.com/hackforla/website/pull/4731#pullrequestreview-1525194986) at 2023-07-11 02:39 PM PDT

**Comment by HackforLABot** [comment]
## Activity Log: 93Belen
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- 93Belen opened: https://github.com/hackforla/website/issues/4631 at 05/08/2023, 08:35 PM PDT
- 93Belen assigned: https://github.com/hackforla/website/issues/4631 at 05/08/2023, 08:45 PM PDT
- 93Belen assigned: https://github.com/hackforla/website/issues/4473 at 05/09/2023, 07:11 AM PDT
- 93Belen opened: https://github.com/hackforla/website/pull/4643 at 05/09/2023, 07:31 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4473#issuecomment-1540258746 at 05/09/2023, 07:39 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4631#issuecomment-1540266766 at 05/09/2023, 07:44 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4643#issuecomment-1541889009 at 05/10/2023, 03:29 AM PDT
- 93Belen opened: https://github.com/hackforla/website/pull/4646 at 05/10/2023, 03:39 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4646#issuecomment-1541911235 at 05/10/2023, 03:45 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4646#issuecomment-1542705702 at 05/10/2023, 12:37 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4646#issuecomment-1542724514 at 05/10/2023, 12:53 PM PDT
- 93Belen PR was closed: https://github.com/hackforla/website/pull/4643#event-9218030767 at 05/10/2023, 10:29 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4473#issuecomment-1544469877 at 05/11/2023, 11:18 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4646#issuecomment-1545490853 at 05/12/2023, 02:58 AM PDT
- 93Belen closed: https://github.com/hackforla/website/pull/4646#event-9239125435 at 05/13/2023, 12:44 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4646#issuecomment-1546740008 at 05/13/2023, 12:57 PM PDT
- 93Belen opened: https://github.com/hackforla/website/pull/4666 at 05/14/2023, 10:16 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4666#issuecomment-1546951524 at 05/14/2023, 10:18 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4665#issuecomment-1546953338 at 05/14/2023, 10:25 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4665#pullrequestreview-1425597241 at 05/14/2023, 01:23 PM PDT
- 93Belen assigned: https://github.com/hackforla/website/issues/4642 at 05/15/2023, 01:16 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4642#issuecomment-1548517786 at 05/15/2023, 01:18 PM PDT
- 93Belen opened: https://github.com/hackforla/website/pull/4675 at 05/15/2023, 01:32 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4675#issuecomment-1548535428 at 05/15/2023, 01:33 PM PDT
- 93Belen closed: https://github.com/hackforla/website/pull/4675#event-9250866880 at 05/15/2023, 01:34 PM PDT
- 93Belen opened: https://github.com/hackforla/website/pull/4677 at 05/15/2023, 01:56 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4677#issuecomment-1548564909 at 05/15/2023, 01:57 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4666#issuecomment-1549571222 at 05/16/2023, 05:29 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4674#issuecomment-1549677168 at 05/16/2023, 06:27 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4674#pullrequestreview-1428600446 at 05/16/2023, 06:34 AM PDT
- 93Belen PR was merged: https://github.com/hackforla/website/pull/4677#event-9263804513 at 05/16/2023, 06:43 PM PDT
- 93Belen issue closed by PR 4677: https://github.com/hackforla/website/issues/4642#event-9263804602 at 05/16/2023, 06:43 PM PDT
- 93Belen PR was merged: https://github.com/hackforla/website/pull/4666#event-9263822406 at 05/16/2023, 06:47 PM PDT
- 93Belen issue closed by PR 4666: https://github.com/hackforla/website/issues/4473#event-9263822503 at 05/16/2023, 06:48 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4674#issuecomment-1550593266 at 05/16/2023, 07:22 PM PDT
- 93Belen assigned: https://github.com/hackforla/website/issues/4687 at 05/17/2023, 03:43 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4687#issuecomment-1552179712 at 05/17/2023, 03:45 PM PDT
- 93Belen opened: https://github.com/hackforla/website/pull/4691 at 05/18/2023, 03:58 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4689#issuecomment-1552899785 at 05/18/2023, 04:09 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4689#pullrequestreview-1432518804 at 05/18/2023, 04:21 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4691#issuecomment-1553520661 at 05/18/2023, 12:13 PM PDT
- 93Belen PR was merged: https://github.com/hackforla/website/pull/4691#event-9286175614 at 05/18/2023, 11:30 PM PDT
- 93Belen issue closed by PR 4691: https://github.com/hackforla/website/issues/4687#event-9286175735 at 05/18/2023, 11:30 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4631#issuecomment-1555132044 at 05/19/2023, 12:26 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4702#issuecomment-1556227024 at 05/21/2023, 10:07 AM PDT
- 93Belen assigned: https://github.com/hackforla/website/issues/4662 at 05/22/2023, 04:19 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1557040450 at 05/22/2023, 04:20 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4702#pullrequestreview-1439648024 at 05/23/2023, 06:00 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1559601481 at 05/23/2023, 07:54 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1559621561 at 05/23/2023, 08:01 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4702#issuecomment-1560011712 at 05/23/2023, 12:28 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1560377117 at 05/23/2023, 07:37 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4710#issuecomment-1560379555 at 05/23/2023, 07:41 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4702#pullrequestreview-1444004597 at 05/25/2023, 07:16 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4710#pullrequestreview-1444081720 at 05/25/2023, 07:51 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1564204141 at 05/26/2023, 03:49 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1564824546 at 05/26/2023, 12:19 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4746#issuecomment-1564979890 at 05/26/2023, 02:29 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4746#pullrequestreview-1447000349 at 05/26/2023, 02:45 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4746#pullrequestreview-1447027631 at 05/26/2023, 03:35 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1565336598 at 05/27/2023, 03:27 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4731#issuecomment-1566214809 at 05/28/2023, 11:05 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4732#issuecomment-1566214953 at 05/28/2023, 11:05 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4731#pullrequestreview-1448373188 at 05/28/2023, 12:01 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4731#issuecomment-1566231274 at 05/28/2023, 12:03 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4732#issuecomment-1567291890 at 05/29/2023, 08:46 AM PDT
- 93Belen issue closed as completed: https://github.com/hackforla/website/issues/4662#event-9372684446 at 05/29/2023, 06:03 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4732#pullrequestreview-1450918277 at 05/30/2023, 06:18 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4731#issuecomment-1568551575 at 05/30/2023, 07:37 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4754#issuecomment-1568557089 at 05/30/2023, 07:40 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4754#pullrequestreview-1451144991 at 05/30/2023, 07:46 AM PDT
- 93Belen closed as completed: https://github.com/hackforla/website/issues/4631#event-9383291032 at 05/30/2023, 02:48 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1569409357 at 05/30/2023, 07:34 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4755#pullrequestreview-1454219190 at 05/31/2023, 02:41 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4759#issuecomment-1571031533 at 05/31/2023, 03:11 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4759#pullrequestreview-1455548062 at 06/01/2023, 06:57 AM PDT
- 93Belen assigned: https://github.com/hackforla/website/issues/4525 at 06/04/2023, 09:38 AM PDT
- 93Belen unassigned: https://github.com/hackforla/website/issues/4525 at 06/04/2023, 09:46 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4755#pullrequestreview-1465261893 at 06/06/2023, 06:49 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4783#issuecomment-1578838212 at 06/06/2023, 07:07 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4755#pullrequestreview-1466176435 at 06/06/2023, 02:52 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4783#pullrequestreview-1466297720 at 06/06/2023, 04:35 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1579601707 at 06/06/2023, 04:39 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4755#pullrequestreview-1466301154 at 06/06/2023, 04:40 PM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1579603412 at 06/06/2023, 04:41 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4783#pullrequestreview-1468533228 at 06/07/2023, 01:43 PM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4783#pullrequestreview-1474827231 at 06/12/2023, 05:36 AM PDT
- 93Belen commented on: https://github.com/hackforla/website/pull/4731#issuecomment-1627773113 at 07/09/2023, 09:59 AM PDT
- 93Belen submitted review: https://github.com/hackforla/website/pull/4731#pullrequestreview-1525194986 at 07/11/2023, 02:39 PM PDT
